features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -104,11 +124,54 @@ The PSA on single v7-M is supported with the help of Mbed OS PSA services. The f
 Check [extra_labels](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html), [device_has](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html) and [features](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html) for more information.
 
-### Adding dual v7-M targets
+## Adding dual v7-M targets
+A target can ge categorized as a dual v7-M target if it has at least two Armv7-M cores. On dual v7-M targets, TF-M will be running on SPE providing PSA services. Therefore, it is **MANDATORY** to define both SPE and NSPE target for dual v7-M targets.
+
+The SPE target **MUST** contain following attributes,
+
+* inherits: PSA generic target PSA_DUAL_V7_M_SPE
+* tfm_target_name: Target name in TF-M
+* tfm_bootloader_supported: If TF-M bootloader is supported by the target. Values supported are "true" and "false"
+* tfm_default_toolchain: Default TF-M toolchain supported. Values supported are "ARMCLANG" and "GNUARM"
+* tfm_supported_toolchains: Supported TF-M toolchains. Values supported are "ARMCLANG" and "GNUARM"
+* delivery_dir: The directory to which TF-M binary will be copied to
+
+The following is taken from `targets/targets.json` and shows a PSA enabled dual v7-M target, `PSoC64`,
+
+```json
+    "CY8CPROTO_064_SB_S": {
+        "inherits": ["PSA_DUAL_V7_M_SPE"],
+        "tfm_target_name": "psoc64",
+        "tfm_bootloader_supported": false,
+        "tfm_default_toolchain": "GNUARM",
+        "tfm_supported_toolchains": ["ARMCLANG", "GNUARM"],
+        "delivery_dir": "TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/prebuilt"
+    },
+    "CY8CPROTO_064_SB_NS": {
+        "inherits": ["PSA_DUAL_V7_M_NSPE", "MCU_PSOC6_M4"],
+        "components_remove": ["QSPIF"],
+        "device_has_remove": ["QSPI"],
+        "supported_form_factors": ["ARDUINO"],
+        "extra_labels_add": ["PSOC6_01", "MXCRYPTO_01"],
+        "macros_add": ["CYB06447BZI_D54",
+                       "PSOC6_DYNSRM_DISABLE=1",
+                       "CY_CFG_SYSCLK_WCO_ENABLED=1",
+                       "SEMAPHORE"],
+        "detect_code": ["1907"],
+        "forced_reset_timeout": 5,
+        "reset_method": "default",
+        "post_binary_hook": {
+            "function": "PSOC6Code.sign_image"
+        }
+    },
+```
+
+Please pay attention to config options `extra_labels_add` and `device_has_remove`. If needed then a PSA target definition **MUST** use [extra_labels/device_has]`_add` or [extra_labels/device_has]`_remove` (not `extra_labels` or `device_has`) to add/remove either extra_labels or target capabilities. Also, use `[feature_]`add/remove to add/remove a feature.
```

Comment:
```diff
- Please pay attention to config options
+ Please pay attention to the config options
```

Reply:
```diff
- If needed then a PSA
+ If needed, a PSA
```

Reply:
Is "`[feature_]`add/remove" supposed to be "`feature_`[add/remove]"?

Reply:
Fixed